mac-os-build/Vagrantfile

```diff
@@ -0,0 +1,39 @@
+PROVISION = <<-SCRIPT
+sudo apt-get update -y
+sudo apt-get install apt-transport-https \
+  ca-certificates \
+  curl \
+  gnupg2 \
+  git \
+  software-properties-common \
+  rsync -y
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+sudo apt-get update -y
+sudo apt-get install -y docker-ce
+[ -d /home/vagrant/.ssh ] && echo '%{public_key}' >> /home/vagrant/.ssh/authorized_keys; true
+[ -d /home/vagrant/.ssh ] && chmod 600 /home/vagrant/.ssh/authorized_keys; true
+sudo usermod -aG docker vagrant
+git clone -b #{ENV['BRANCH']} --single-branch https://github.com/vyos/vyos-build /opt/vyos-build
+docker run --rm -t --privileged -v /opt/vyos-build:/vyos -w /vyos vyos/vyos-build:#{ENV['BRANCH']} \
+  /bin/bash -c './configure --architecture amd64 --build-by "hackerman@vyos.io" && make iso'
```

Comment:
There should rather be a way to pass configure options and make target through environment variables.

Reply:
Will do that, great suggestion